### PR TITLE
New profileEditEmailPage #116 

### PR DESCRIPTION
**Pull request by olavea**

### CODE REVIEW

**Comment by olavea** [comment]
I imported ProfileEditPage into ProfileIndexPage
I imported ProfileIndexPage into pages/profile.js

File: src/components/ProfileEditPage.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import ProfileOrder from "./ProfileOrder"
+import ProfileInventory from "./ProfileInventory"
+
+class ProfileEditPage extends React.Component {
+  render() {
+    return (
+      <form className="email-edit">
+        <input type="text" required placeholder="🦄@usepow.app" />
+        <button type="submit">Update</button>
+        <ProfileHeader />
```
Comment (raae) [inline src/components/ProfileEditPage.js:13]: Please stay on task and remove these.

File: src/pages/profile.js [hunk 2]
```diff
@@ -32,16 +34,20 @@ const ProfilePage = () => {
...
-      </>
+      </div>
     )
   }
 
   return (
     <CycleProvider entries={entries} settings={settings}>
       <SEO title="Profile" />
+      <Router basepath="/profile">
+        <ProfileIndexPage path="/edit" />
```
[the remark this answers is not in the document]
Comment (olavea) [reply src/pages/profile.js]: This commit 5e2023e

File: src/components/ProfileIndexPage.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import React from "react"
+import ProfileEditPage from "./ProfileEditPage"
+
+const ProfileIndexPage = () => {
+  return <ProfileEditPage />
```
[the remark this answers is not in the document]
Comment (olavea) [reply src/components/ProfileIndexPage.js]: These commits 
2a7f6b0

"and deleted in profile.js #116"
f042e79

File: src/components/ProfileEditPage.js (hunk 1 above)
Comment (olavea) [reply src/components/ProfileEditPage.js]: Ok 😸 👍

File: src/components/ProfileEditPage.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
+import React from "react"
+// import Header from "./Header"
+import ProfileHeader from "./ProfileHeader"
+import ProfileOrder from "./ProfileOrder"
+import ProfileInventory from "./ProfileInventory"
+
+class ProfileEditPage extends React.Component {
```
[the remark this answers is not in the document]
Comment (raae) [reply src/components/ProfileEditPage.js:11]: I see you are continuing with wesbos classes, the code seems a lot more complicated than it needs to be. Would you like me to make a video? Or example in Code Sandbox @olavea

**Comment by raae** [comment]
I could move back and forth, thanks!

It would also be nice with a "back" link above the input fields for moving back to the profile page without using the update button.

**Comment by olavea** [comment]
Create a place to hold the email with useState.
- [ ] const [email, setEmail] = useState("") initializing the e-mail to an empty string
- [ ]  Upated the email using setEmail when the value in the input changes (wesbos probably uses setState, but you should use setEmail).

**Comment by raae** [comment]
The user story we are trying to solve: 
- As a logged in user I should be able to change my email address

Steps taken by user:
- [x] User selects edit
   - Already completed
- User inputs new e-mail address submits the form
   - [ ] Follow video about creating a basic form that will result in an alert with the user's email
       - https://egghead.io/lessons/react-make-basic-forms-with-react-601487f6
   - [ ] Swap show alert with updating the userbase user using `updateUser` from `useAuthActions`.
       - I will make short video on this.

**Comment by raae** [comment]
Code looking good.

**Comment by olavea** [comment]
1. Prevent that form from naughtily self-submitting

2. Get that email from the input
c4b1c96
POW-profile_edit_flow_2  Get that email from the input_2-187

3. Send that email to Daniel V.'s Userbase
0829034

POW-profile_edit_flow_3  Send that email to Daniel V 's Userbase_3-188


4. Send that customer back to /profile
3b21518

POW-profile_edit_flow_3  4  Send that customer back to :profile_4-189

**Comment by raae** [comment]
The code looks good, before merging everything needs to be cleaned up.

Files that are not used must be deleted, imports that are not used must be deleted and linting errors must be fixed.

I will do a last styling update that I record after that, then you can merge.

**Comment by olavea** [comment]
warning A form label must be associated with a control jsx-a11y/label-has-associated-control
https://github.com/airbnb/javascript/issues/1885

**Comment by raae** [comment]
I will run this and do some styling fixes tomorrow while recording, but it's looking good!

**Comment by olavea** [comment]
GLEE 0aa9189

G
L
E
E

**Comment by raae** [comment]
You should test the current code on both mobile and big screen. If you are happy you may squash and merge. 

Please rename the title to "Add edit email feature  (#125)" and add fitting bullet points in the description. There are way too many commits to include them all.

**Comment by olavea** [comment]
Questions:
One <toolbar /> is hidden under another <toolbar /> and <AppBar />?

File: src/components/ProfileEditPage.js [hunk 5]
```diff
@@ -28,32 +59,41 @@ const ProfileEditPage = () => {
...
-              />
-            </label>
-          </div>
+    <div className={classes.root}>
+      <form
+        onSubmit={createEmail}
+        onReset={createReset}
+        className={classes.form}
+      >
+        <Toolbar />
```
Comment (olavea) [inline src/components/ProfileEditPage.js]: This` <toolbar /> `is hidden under 
the` <toolbar /> `in line 96
and the` <AppBar /> `in line 90?

File: src/components/ProfileCard.js [hunk 6]
```diff
@@ -32,6 +38,18 @@ const ProfileCard = () => {
     updateUser({ email: userEmail, profile: { ...userProfile, ...profile } })
   }
 
+  // Open / close menu
+
```
Comment (olavea) [inline src/components/ProfileCard.js:42]: What is happening in this useState?

Set the new email as email in newsletter?

**Pull request merged**